scripts/generate_builtins_dafsa.zig

```diff
@@ -0,0 +1,945 @@
+// Generates a DAFSA and a companion array of builtin properties from the set of .def
+// files from Clang found here:
+//
+// https://github.com/llvm/llvm-project/tree/main/clang/include/clang/Basic
+//
+// To generate, run the following command from the root of the repository:
+//
+// zig run scripts/generate_builtins_dafsa.zig --main-pkg-path . -- path/to/Builtins*.def
```

Comment:
```suggestion
// zig run scripts/generate_builtins_dafsa.zig --main-mod-path . -- path/to/Builtins*.def
```
This changed recently.

Reply:
Ah, was on an older version due to https://github.com/ziglang/zig/issues/17482